zerver/tests/test_message_send.py

```diff
@@ -3358,7 +3342,7 @@ def test_no_topic_message(self) -> None:
         sender = self.example_user("iago")
         client = make_client(name="test suite")
         stream = get_stream("Denmark", realm)
-        topic_name = "(no topic)"
+        topic_name = ""
```

Comment:
As noted above, we probably want to split this into two tests, one for "(no topic)" legacy behavior, and one for the new `""`.

Reply:
Done